.gitignore

```diff
@@ -0,0 +1 @@
+secrets.env
```

Comment:
You should probably add a `.dockerignore` with the same contents, to keep your secret file from being copied in.